bin/main/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```

Comment:
What is the function of this file, there is already a copy of it in `src/main/resources/META-INF` Is there a different function for this file?